internal/database/badger/writebatch.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package badger
+
+import "github.com/dgraph-io/badger/v3"
+
+// writeBatch uses the badger write batch and prefixes
+// all keys with a certain given prefix.
+type writeBatch struct {
```

Comment:
what uses this?

Reply:
The database `NewWriteBatch()` method and the database's table method `NewWriteBatch()`